content/cluster/_index.md

```diff
@@ -1,10 +1,71 @@
 ---
 title: "Cluster"
-date: 2019-06-18T16:42:20+02:00
-lastmod: 2019-06-18T16:42:20+02:00
-publishdate: 2019-06-18T16:42:20+02:00
+date: 2020-04-21T15:22:20+02:00
+lastmod: 2020-04-21T15:22:20+02:00
+publishdate: 2020-04-21T15:30:20+02:00
 draft: false
 weight: 13
 ---
 
-Nothing to see here (yet)
+# Move from control-plane cluster to a control-plane and compute nodes cluster
+
+## Create computes
+
+```		
+oc label nodes ocp4-worker-0.example.com node-role.kubernetes.io/infra="" 
+oc label nodes ocp4-worker-1.example.com node-role.kubernetes.io/infra=""
+oc get nodes
+```
```

Comment:
shouldn't you remove the node-role.kubernetes.io/worker="" label from the masters?

Reply:
It isn't mandatory but I'll add a note like:
Note: you may want to remove the node-role.kubernetes.io="" label from the masters, up to you.